docs/source/upcoming_release_notes/1292-recoving_sp1k4-att_targets.rst

```diff
@@ -0,0 +1,30 @@
+1292 recoving sp1k4-att targets
+#################
+
+API Breaks
+----------
+- N/A
+
+Library Features
+----------------
+- N/A
+
+Device Features
+---------------
+- recover target number from 3 to 5
```

Comment:
I think this doc shouldn't pass review, it will be missing context when combined with the other release notes. I'll try to remember that this one comes from sp1k4 when we do the pcdsdevices tag.

Reply:
Thanks Zach. I can upload again for more details" recover sp1k4-att target number from 3 to 5".

Reply:
whoops sorry, I was focused on the code changes.  That's my bad